src/ResourceManager/IotHub/Commands.IotHub/help/Add-AzureRmIotHubCertificate.md

```diff
@@ -79,7 +79,7 @@ Updates a CA certificate in an IoT hub by uploading a new CER file.
 Name of the Certificate
 
 ```yaml
-Type: System.String
+Type: String
```

Comment:
You need to run your help generation with the `-UseFullTypeName` parameter. It will remove all these changes where the whole typename is being removed. See:
https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module

Reply:
Done